s3transfer/manager.py

```diff
@@ -27,7 +28,9 @@ def __init__(self,
                  multipart_threshold=8 * MB,
                  max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0):
+                 max_queue_size=0,
```

Comment:
I agree that smaller stream sizes are better as long as it doesn't have any perf differences from larger chunk sizes.  Given this is all IO bound, I wouldn't expect there to be any differences, but it's worth verifying.